library/objective-c/BUILD

```diff
@@ -32,6 +31,15 @@ objc_library(
     ],
 )
 
+objc_library(
```

Comment:
Should this be added as a dependency somewhere?

Reply:
(was wip) :)